integration-tests/load-tests/load-test-helpers.ts

```diff
@@ -0,0 +1,228 @@
+/* Imports: External */
+import { ethers } from 'ethers'
+
+/* Imports: Internal */
+import { OptimismEnv } from '../test/shared/env'
+import { sleep } from '../test/shared/utils'
+import { Direction } from '../test/shared/watcher-utils'
+
+interface TransactionParams {
+  contract: ethers.Contract
+  functionName: string
+  functionParams: any[]
+}
+
+export const executeL1ToL2Transactions = async (
+  env: OptimismEnv,
+  txs: TransactionParams[]
+) => {
+  for (const tx of txs) {
+    const signer = ethers.Wallet.createRandom().connect(env.l1Wallet.provider)
+    const receipt = await env.l1Messenger
+      .connect(signer)
+      .sendMessage(
+        tx.contract.address,
+        tx.contract.interface.encodeFunctionData(
+          tx.functionName,
+          tx.functionParams
+        ),
+        8000000,
+        {
+          gasPrice: 0,
+        }
+      )
+
+    await env.waitForXDomainTransaction(receipt, Direction.L1ToL2)
+  }
+}
+
+export const executeL2ToL1Transactions = async (
+  env: OptimismEnv,
+  txs: TransactionParams[]
+) => {
+  for (const tx of txs) {
+    const signer = ethers.Wallet.createRandom().connect(env.l2Wallet.provider)
+    const receipt = await env.l2Messenger
+      .connect(signer)
+      .sendMessage(
+        tx.contract.address,
+        tx.contract.interface.encodeFunctionData(
+          tx.functionName,
+          tx.functionParams
+        ),
+        8000000,
+        {
+          gasPrice: 0,
+        }
```

Comment:
Is it possible to run these tests with nonzero gas price? Perhaps this could be an extension issue to improve these tests.

Reply:
Not yet :-/ definitely could be an extension issue though.